### ISSUE TITLE
Change PHP version?

### ISSUE THREAD

**Issue by killerb77** [nextcloud/docker#1858]
Is it possible to change the PHP version from within the docker container? Specifically I want to change Nextcloud 25 from PHP 8.1 to PHP 8.0.

**Comment by martadinata666** [comment]
yes you can, assuming you know how to rebuild an image, you can clone the repo, go to `25` folder replace the `FROM` part.

**Comment by killerb77** [comment]
@martadinata666 That worked.  Thank you.  Was actually easier than I thought it would be.

![Screenshot 2022-11-07 192950](https://user-images.githubusercontent.com/56096557/200444606-7ad93c49-39f8-4392-8183-72629f790400.jpg)
